openstack/neutron/values.yaml

```diff
@@ -496,6 +496,30 @@ mysql_metrics:
         FROM networks;
       values:
       - "value"
+    - help: Neutron total segments per hostgroup
+      labels:
+      - "host"
+      name: openstack_neutron_total_network_segments
+      query: |
+        SELECT
+          COUNT(*) AS value
```

Comment:
Don't we have to select the `host` if we want it as a label?

Reply:
Thanks, yes